src/main/java/seedu/academydirectory/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,8 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_STUDIO_SESSION = new Prefix("ses/");
+    public static final Prefix PREFIX_STUDIO_ATTENDANCE = new Prefix("att/");
+
```

Comment:
Speaking of which we might want to reconsider how we set the prefix tbh. I find it a bit hard to memorize even the previous AB3 prefixes.

Reply:
Thanks for the catch! Caused by a merge conflict I forgot to resolve. Will fix in next iteration. :)